app-misc/piper/piper-0.7.ebuild

```
@@ -3,27 +3,38 @@
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{8..10} )
+PYTHON_COMPAT=( python3_{8..11} )
 
 inherit meson python-single-r1 xdg
 
 DESCRIPTION="GTK configuration application for libratbag"
```

The "official" description on https://github.com/libratbag/piper reads

> GTK application to configure gaming devices

which is more informative, as most people will have no idea what libratbag is.

```suggestion
DESCRIPTION="GTK application to configure gaming devices"
```

---

Thank you! I agree with you, this is way more informative